database/layer/admin-persona.js

```diff
@@ -1,13 +1,9 @@
 const { PGPool } = require('./pg_pool')
 const pool = new PGPool()
+const { usherDb } = require('../../database/layer/knex')
```

Comment:
@k1ch  how come you start the require with `../..` ? Why did you not just do `require('./knex')`

Reply:
Nice catch! Probably I didn't pay attention to the autofill suggestion.